components/basic-analyzers/src/main/java/org/datacleaner/beans/CompletenessAnalyzer.java

```diff
@@ -191,7 +215,7 @@ public void fillAllConditions(Condition condition) {
         final OutputDataStreamBuilder incompleteStreamBuilder = OutputDataStreams
                 .pushDataStream(OUTPUT_STREAM_INCOMPLETE);
 
-        for (InputColumn<?> column : _valueColumns) {
+        for (InputColumn<?> column : createOutputDataStreamColumns()) {
```

Comment:
Is that necessary to create them here instead of iterating _outputDataStreamsColumns? I thought they are created once in the init method.